### DISCUSSION TITLE
javascript on rigtools

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#1252] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so does anyone have anything that can do somthing cool on rig
![Screenshot 2024-11-08 12 58 24 PM](https://github.com/user-attachments/assets/cd2ad20a-dfcc-4bfb-8cfb-e49422b1d0bd)
tools

**Comment by Sam-hacks1** [comment]
how do you even get onto that screen. When I do it, it just shows a bunch of java scirpts

**Reply by nullkid10** [reply]
You’re on version 129 or later. Rigtools isn’t supported on version 129 or later.

**Comment by ghost** [comment, marked as answer]
if you enable eruda or chii and create a new tab you should screen in the bottom or a thing in the right corner. you can run some javascript in the console.

**Reply by Balistic123** [reply]
how would i do that?

**Comment by BobertoBeans** [comment]
Why does your thing look like that? Mine looks like this:
![Screenshot 2025-02-02 5 12 50 AM](https://github.com/user-attachments/assets/fc0e61d3-0f30-43c7-9894-d4fa26f272a5)

**Reply by S-PScripts** [reply]
they were using the old gui i presume
